hipparchus-core/src/changes/changes.xml

```diff
@@ -49,6 +49,11 @@ If the output is not quite correct, check for invisible trailing spaces!
     <title>Hipparchus Core Release Notes</title>
   </properties>
   <body>
+    <release version="?" date="TBD" description="">
+      <action dev="vcucchie and luc" type="add" issue="issues/218">
```

Comment:
you can remove my name here, I didn't do anything!

Reply:
Now you did :smile: !